certs/certs.go

```diff
@@ -64,6 +66,7 @@ func WriteNewCerts(certFile, keyFile string, server bool) error {
 		KeyUsage:              x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature | x509.KeyUsageCertSign,
 		BasicConstraintsValid: true,
 		DNSNames:              []string{"localhost"},
+		IPAddresses:           []net.IP{net.IPv4(127, 0, 0, 1)},
```

Comment:
We can save this for later... someone else recently introduced config.Host, and I think there's still a couple things to hash out to make it work well, this code included.  Don't want to have that stop you from getting this in, though.